### PR TITLE
fix: update econnreset description

### LINKED ISSUE #523: Update ProtonVPN information to accommodate for free plan changes
### Bug type

Wrong data or information

### Describe the bug

The United States is now available only with a paid subscription. I suggest removing the country specific remark on the bot message and replacing it with something more generic. Solving ECONNRESET issues seems to work just as well when connecting to other countries.

**Edit**: The United States is available in the free plan but the location you will end up with is very random since it's up to Proton to decide what's better.

### Expected behavior

Updated command information

### Steps to reproduce

N/A

### References (optional)

image
image

### Additional info (optional)

_No response_

### Discord Username (optional)

ccute

### PR DESCRIPTION
**Pull request by LunakisDev**
Fixes #523
## Description
This PR change `.econreset` | `.econnreset` description:

From: `To verify that this issue is not caused by your Internet Service Provider, please use a VPN service with the connection location set to the US and try again.`

To: `To verify that this issue is not caused by your Internet Service Provider, please use a VPN service and try again.`

## Test Results

image

## Discord Username
`LunakisLeaks`

### CODE REVIEW

**Review by ExampleWasTaken — COMMENTED** [review]
LGTM

But we may want to wait for a final decision regarding the discussion in #523.

**Review by Kostiskat — APPROVED** [review]
LGTM - I will forward this for internal discussion with the crew.

**Review by benw202 — APPROVED** [review]

**Pull request merged**
